### PR TITLE
Leave mpe room client

### PR DESCRIPTION
**Pull request by prastoin**
related #107 
still need to finish e2e tests

### CODE REVIEW

**Comment by prastoin** [comment]
Setting as draft as:
After a client received a forced disconnection, atm he will always be redirected the user to the library. A user can receive a forced_disconnection at any moment. We want him to be redirected only if the user was in disconnected from mpe room view
TODO and tests

File: packages/client/machines/appMusicPlaylistsMachine.ts [hunk 1]
```diff
@@ -60,11 +62,32 @@ export const appMusicPlaylistsModel = createModel(
...
+            LEAVE_ROOM: (args: { roomID: string }) => args,
+            RECEIVED_FORCED_DISCONNECTION: (args: {
+                roomSummary: MpeRoomSummary;
+            }) => args,
+            RECEIVED_MPE_LEAVE_ROOM_CALLBACK: (args: {
+                roomSummary: MpeRoomSummary;
+            }) => args,
+            SPAWN_NEW_PLAYLIST_ACTOR_FROM_STATE: (args: {
+                state: MpeWorkflowState;
+            }) => args,
```
Comment (Devessier) [inline packages/client/machines/appMusicPlaylistsMachine.ts:74]: This event can be safely deleted as it has been replaced by `ROOM_CREATION_ACKNOWLEDGEMENT` in https://github.com/AdonisEnProvence/MusicRoom/pull/359

File: packages/client/machines/appMusicPlaylistsMachine.ts [hunk 2]
```diff
@@ -114,7 +137,9 @@ export const appMusicPlaylistsModel = createModel(
             ) => args,
             ///
 
-            REMOVE_ACTOR_FROM_PLAYLIST_LIST: (args: { roomID: string }) => args,
+            STOP_AND_REMOVE_ACTOR_FROM_PLAYLIST_LIST: (args: {
+                roomID: string;
+            }) => args,
```
Comment (Devessier) [inline packages/client/machines/appMusicPlaylistsMachine.ts:142]: I have to say that this event is really interesting Whenever you want to (stop and) remove an actor, you send the event to the machine itself. This is really elegant.

**Review by Devessier — DISMISSED** [review]
So great!
We can merge.

File: packages/client/machines/appMusicPlaylistsMachine.ts (hunk 1 above)
Comment (prastoin) [reply packages/client/machines/appMusicPlaylistsMachine.ts:74]: Thanks, merge conflict mistake I guess

**Review by Devessier — APPROVED** [review]
Perfect!
We can merge.

**Pull request merged**